common/scripts/report_build_info.sh

```diff
@@ -44,3 +44,4 @@ echo "istio.io/pkg/version.buildGitRevision=${BUILD_GIT_REVISION}"
 echo "istio.io/pkg/version.buildStatus=${tree_status}"
 echo "istio.io/pkg/version.buildTag=${GIT_DESCRIBE_TAG}"
 echo "istio.io/pkg/version.buildHub=${HUB}"
+echo "istio.io/pkg/version.buildArch=${TARGET_ARCH}"
```

Comment:
Files in the `common` directory come from istio/common-files. This change would need to be made in that repo and when committed, it will propagate to the other repos like here.

Reply:
@ericvn Thanks